services/download/src/main/java/app/coronawarn/server/services/download/BatchDownloadException.java

```diff
@@ -0,0 +1,14 @@
+
+
```

Comment:
please remove empty lines at the beginning of the file - the package declaration should be the first statement